doc/policies/elemtype.hpp

```diff
@@ -1,6 +1,6 @@
 /*! @page elem The ElemType policy in mlpack
 
-@section Overview
+@section ElemTypeOverview
```

Comment:
```suggestion
@section elem_overview Overview
```

I think this one was just missing a section name.